frontend/src/pull-card/index.tsx

```diff
@@ -103,6 +105,22 @@ export const PullCard = memo(function PullCard({
           />
           <Flags pull={pull} />
         </SigsAndFlags>
+        {showLinesChanged &&
+        <Code backgroundColor="transparent">
+          <HStack>
+            <Text fontFamily={'Verdana'}
+              title={pull.additions == 1 ? `${pull.additions} addition` : `${pull.additions} additions`}
+              fontSize={"12px"}
```

Comment:
For what it's worth, if you're setting a prop to a static string, you don't need curly braces. Curly braces say "evalute and return this javascript expression". IF your expression is just a static string you don't need the JS eval part: `fontSize="12px"`.

Reply:
Thank you, I'm getting a bit more used to using React so that is helpful

Reply:
Resolved via https://github.com/iFixit/pulldasher/pull/365/commits/80657a6243c41fc7568a846fd91d48205454eabb